database/statusDataManager.ts

```diff
@@ -0,0 +1,28 @@
+import { DB_CLIENT } from "./dbConfig";
+
+const STATUS_TABLE: string = "boba-buddies-status";
```

Comment:
i made a table in DynamoDB that literally has 1 row that is updated every time `putStatus` is called LMFAO